bin/build.sh

```diff
@@ -130,13 +130,13 @@ if [[ "${build}" = true || "${build_js}" = true ]]; then
       sudo gem install compass
     fi
   fi
-  (cd ./modules/ui && npm install && bower install && grunt build)
+  (cd ./modules/ui && npm install && bower install && grunt --forcce build)
```

Comment:
Double 'c' in `forcce`. Not sure if you need it at all. Also why is the need to add /target/ ?